guides/v2.0/cloud/live/stage-prod-migrate.md

```diff
@@ -161,26 +159,31 @@ To migrate a database:
 	For Pro Staging and Production environments, the name of the database is in the `MAGENTO_CLOUD_RELATIONSHIPS` variable (typically the same as the application name and user name):
 
 		mysqldump -h <database host> --user=<database user name> --password=<password> --single-transaction --triggers <database name> | gzip - > /tmp/database.sql.gz
```

Comment:
We should probably recommend using `vendor/bin/ece-tools db-dump` here instead.  That way you don't have to lookup the username, databasename, hostname, and password.  Though, I did recently file a bug report on our tool because one of the options we are using fails on large database tables, so we may want to wait on changing this line of devdocs.  https://magento2.atlassian.net/browse/MAGECLOUD-2215

Reply:
vendor/bin/ece-tools db-dump might not have DEFINER replaced, this is being reviewed by @yyevgenii

Reply:
That is true - it does not.  We should probably add that as an option to `ece-tools db-dump` and may even want to make it as a default option because I see people running into it a lot recently.